contrib/test_jenkins.sh

```diff
@@ -180,6 +180,47 @@ get_active_ib_devices() {
 	done
 }
 
+#
+# Get list of active IP interfaces
+#
+get_active_ip_ifaces() {
+	echo $(ip addr | awk '/state UP/ {print $2}' | sed s/://)
+}
+
+#
+# Get IP addr for a given IP iface
+# Argument is the IP iface
+#
+get_ifaddr() {
+	iface=$1
+	echo $(ip addr show ${iface} | awk '/inet /{print $2}' | awk -F '/' '{print $1}')
+}
+
+get_ipoib_address() {
+	iface=`ibdev2netdev | grep Up | awk '{print $5}' | head -1`
+	if [ -n "$iface" ]
+	then
+		ipaddr=$(get_ifaddr ${iface})
+	fi
+
+	if [ -z "$ipaddr" ]
+	then
+		# if there is no inet (IPv4) address, escape
+		echo ""
```

Comment:
do we really need this?

Reply:
removed, but need to wrap a result by ""

Reply:
10x, BTW could use `if [ -z $server_addr ]`

Reply:
@yosefe yes, right. Thanks for the hint!